app/konnect/configure/runtime-manager/runtime-groups/declarative-config.md

```diff
@@ -43,34 +54,47 @@ Successfully Konnected as Some Name (Kong)!
 
 You can also use decK with {{site.konnect_short_name}} more securely by storing
 your password in a file, then either calling it with
-`--konnect-password-file <pass>.txt`, or adding it to your decK configuration
-under the `konnect-password` option.
+`--konnect-password-file {FILENAME}.txt`, or adding it to your decK configuration
+under the `konnect-password` option along with your email:
 
-The following steps all use `--konnect-password-file`.
+```yaml
+konnect-password: {YOUR_PASSWORD}
+konnect-email: {YOUR_EMAIL}
+```
 
+The following steps all use a `deck.yaml` file to store the
+{{site.konnect_short_name}} credentials instead of flags.
 
 ## Create a configuration file
 
 Capture a snapshot of the current configuration in a file:
 
 ```sh
-deck konnect dump --konnect-email <email> --konnect-password-file <pass>.txt
+deck dump --konnect-runtime-group default
 ```
 
-The command creates a file named `konnect.yaml`. If you have nothing
-configured, decK creates the file with only the format version:
+If you don't specify the `--konnect-runtime-group` flag, decK still targets the
+`default` runtime group. If you have more than one runtime group in your
+organization, we recommend always setting this flag to avoid accidentally
+pushing configuration to the wrong group.
+
+The command creates a file named `kong.yaml`. If you have nothing
+configured, decK creates the file with only the format version and runtime group
+name:
 
 ```yaml
-_format_version: "0.1"
+_format_version: "1.1"
+_konnect:
+  runtime_group_name: default
 ```
 
 You can specify a different file name or location, or export the
 configuration in JSON format:
 
 ```sh
-deck konnect dump --konnect-email <email> --konnect-password-file <pass>.txt \
---format json \
---output-file examples/konnect2.yaml
+deck dump --konnect-runtime-group default \
+  --format json \
+  --output-file examples/konnect.yaml
```

Comment:
Is the mismatch between `--format json` and `.yaml` on purpose?

Reply:
Ah, no. That's a typo 😅